packages/pwa-kit-dev/src/ssr/server/loading-screen/index.html

```diff
@@ -120,5 +120,11 @@ <h1 class="loading-text">Building your app</h1>
             animateFacts()
         })()
     </script>
+
+    <script>
+        (function() {
+            fetch('/__mrt/clear-browser-data')
```

Comment:
Calling this at the end of `<body>` so to avoid delays with the rendering of the loading page. 

I've tried setting the Clear-Site-Data header to the index.html. But that results in a noticeable delay to the page rendering. So I decided to do it this way instead.

Reply:
Maybe to give control back to the developers, we can provide a flag like `npm start:no-clear-browser-data` to bypass it.

Reply:
This must not be the default. Otherwise developers will have no way to test caching.

Reply:
This must not be the default. Otherwise developers will have no way to test caching.

Reply:
It's possible to test caching, while your dev server still runs and not restarted.